src/sentry/runner/commands/devserver.py

```diff
@@ -6,6 +6,25 @@
 
 from sentry.runner.decorators import configuration, log_options
 
+_DEFAULT_DAEMONS = {
+    "worker": ["sentry", "run", "worker", "-c", "1", "--autoreload"],
+    "cron": ["sentry", "run", "cron", "--autoreload"],
+    "post-process-forwarder": [
+        "sentry",
+        "run",
+        "post-process-forwarder",
+        "--loglevel=debug",
+        "--commit-batch-size=1",
+    ],
+    "ingest": ["sentry", "run", "ingest-consumer", "--all-consumer-types"],
+    "server": ["sentry", "run", "web"],
+    "storybook": ["./bin/yarn", "storybook"],
```

Comment:
Similarly here -- I don't know that it makes sense to include a `yarn` command.

Reply:
I don't think we even need the `styleguide` target in devserver. It is simpler to run `yarn storybook` directly imo.